feeds/ucentral/ucentral-schema/Makefile

```diff
@@ -6,9 +6,9 @@ PKG_RELEASE:=1
 PKG_SOURCE_URL=https://github.com/Telecominfraproject/wlan-ucentral-schema.git
 PKG_MIRROR_HASH:=c641622188b9b378550c21b1dcfa105b08a60e534d7d77ae63001d38a3e95cd8
```

Comment:
please drop this patch, I have already updated the main branch to latest ucentral-schema HEAD